### PR TITLE
Poster: Open Classification of Regimes in the Southeast USA (OpenCRUMS USA): An open source framework for classifying aerosol and meteorological regimes in the Southeast USA.

### PR DESCRIPTION
**Pull request by rcjackson**
Hello, I am submitting my virtual poster entitled "Open Classification of Regimes in the Southeast USA (OpenCRUMS USA):
An open source framework for classifying aerosol and meteorological regimes in the Southeast USA." I have attached the PDF and json file for Zenodo. Please let me know if you have any questions.

### CODE REVIEW

File: presentations/posters/robert_jackson/info.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+    "title": "OpenCRUMS: Open Classification of Regimes in the Southeast USA (OpenCrums USA):
```
Comment (deniederhut) [inline presentations/posters/robert_jackson/info.json]: github seems to think this is bad json

Comment (rcjackson) [reply presentations/posters/robert_jackson/info.json]: I've shortened the title which seems to fix this.

Comment (deniederhut) [reply presentations/posters/robert_jackson/info.json]: Thanks!

**Review by deniederhut — APPROVED** [review]
giphy-3548406831

**Pull request merged**